src/components/teampage/team-members/teams-styles.js

```diff
@@ -0,0 +1,75 @@
+import styled from "styled-components"
+import { H2, HUpper } from "../../../constants/typography"
+import { navyBlue, royalBlue } from "../../../constants/colors"
+import { size } from "../../../constants/devices"
+
+const TeamContainer = styled.div`
+  width: 86%;
+  margin: 0 auto;
+  padding-bottom: 16%;
+
+  @media only screen and (max-width: ${size.tablet}px) {
+    padding-bottom: 30vw;
+  }
+  @media only screen and (max-width: ${size.mobileMax}px) {
```

Comment:
I deleted `size.mobileMax` a while back so this line isn't rendering but you can probably add it back in to `devices.js`. You could also use `devices.mobileMax` instead